packages/web-client/app/router.ts

```diff
@@ -16,5 +16,8 @@ Router.map(function () {
   this.route('pay', {
     path: '/pay/:network/:merchant_safe_id',
   });
+  this.route('pay-error', {
```

Comment:
Would it be better to name this `pay-missing-route` or something like that? Just to make it a bit clearer.

Reply:
At the moment this is reusing Ember's error substate route (that is rendered when we fail to fetch a merchant safe) https://guides.emberjs.com/release/routing/loading-and-error-substates/#toc_error-substates, is there a good way to specify that without creating a new route? Should we create a new route for this case?

Reply:
Having a declared route named the same as implicit routes like this should be avoided, IMO

Reply:
You can use the `templateName` property of a route to reuse the template.